internal/encryption/method/aesgcm/example_test.go

```diff
@@ -56,14 +56,13 @@ func Example_config() {
 
 	// Obtain a modifiable, buildable config. Alternatively, you can also use ConfigStruct() method to obtain a
 	// struct you can fill with HCL or JSON tags.
-	config := descriptor.TypedConfig()
+	config := descriptor.ConfigStruct().(*aesgcm.Config)
```

Comment:
I discussed this with Janos thismorning and the reason we had the TypedConfig() method here was to avoid down-casting and we should add this `TypedConfig()` method back in.